src/Symfony/Component/HttpClient/Response/TraceableResponse.php

```diff
@@ -32,57 +34,98 @@ class TraceableResponse implements ResponseInterface, StreamableInterface
     private $client;
     private $response;
     private $content;
+    private $event;
 
-    public function __construct(HttpClientInterface $client, ResponseInterface $response, &$content)
+    public function __construct(HttpClientInterface $client, ResponseInterface $response, &$content, StopwatchEvent $event = null)
     {
         $this->client = $client;
         $this->response = $response;
         $this->content = &$content;
+        $this->event = $event;
+    }
+
+    public function __destruct()
+    {
+        try {
+            unset($this->response);
```

Comment:
This should be `$this->response = null`, not `unset($this->response)`, to avoid unsetting the property itself

Reply:
why not just remove this? it'd be less risky - no need for the try/catch, let's just stop()

Reply:
If we just "stop", we won't show the time elapsed while fetching the response's status.
ie 
```php
$client->request('GET', 'https://httpstat.us/200'); // $response  not affected => immediatly destructed
```
| just stop | after unset |
|-----------|-------------|
| ![127 0 0 1_8000__profiler_latest_panel=time (1)](https://user-images.githubusercontent.com/578547/96986326-c9486100-1521-11eb-871c-b3f862c48a3b.png) | ![127 0 0 1_8000__profiler_latest_panel=time](https://user-images.githubusercontent.com/578547/96986327-c9e0f780-1521-11eb-9f49-d3cf24c524df.png) |


I there a better way to let the response finish without altering the throw mechanism?

Reply:
`$this->response->__destruct()`